### DISCUSSION TITLE
"Copilot assisted" or "Powered by Copilot" badge

### DISCUSSION THREAD

**Discussion by PiotrFLEURY** [community/community#56848] [Copilot Conversations]
First I want to thanks the Github team for giving me a bêta access to the Copilot chat.
This tool is really great and is capable of making wonders.

I was wondering: _Can or should we create a badge or something to indicate that a source code writing has been assisted by Copilot or Copilot chat ?_
I mean, there is a lot of things made by AI on the internet now and it can be difficult to make the difference.

With something that indicates to other developpers that a particular piece of code has been partially or totally generated using AI assistance we can kill two bird with one stone:

1. We can show examples of what can Copilot do
2. We can be honest and tell everyone that code has been provided partially or totally by an AI

**Can it be possible to discuss the possibility to create an official Github open badge, tag or something in order to do say** _"I have been assisted by an AI to write this code"_ **?**

Something like **Copilot assisted** or **powered by Copilot**

**Comment by aristotaloss** [comment]
Are you actually serious?

**Reply by PiotrFLEURY** [reply]
Yes, why not ?

**Reply by aristotaloss** [reply]
Who the hell cares if code was written by hand or not?

**Reply by spawn-guy** [reply]
@aristotaloss would you like to update your message in 2026? 😉

**Reply by aristotaloss** [reply]
> @aristotaloss would you like to update your message in 2026? 😉

No lol

**Comment by SlimeVRX** [comment]
Good idea!
